src/drone.py

```diff
@@ -0,0 +1,171 @@
+import roslibpy
+from abc import ABC, abstractmethod
+from enum import Enum
+
+class Drone(ABC):
+
+    class Flight_Status(Enum):
+        ON_GROUND_STANDBY = 1
+        IN_AIR_STANDBY = 2
+        FLYING = 3
+        FLYING_HOME = 4
+        PAUSED_IN_AIR = 5
+        LANDING = 6
+        NULL = 7
+
+    class UpdateMissionAction(Enum):
```

Comment:
There is no Update Mission method, where should we use this enum?

Reply:
Will add abstract update_mission method, pending future removal if redundant.